Header header
float64 azimuth
float64 elevation
uint16 obj_id
